packages/eslint-config-airbnb-base/flat.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Changes to support the flat config format:
+ * - The `extends` key has been deprecated.
+ * - The `parserOptions` key has been moved under the new `languageOptions` key.
+ * - The `env` key has been deprecated and moved to `languageOptions.globals`.
+ * - The `plugins` key is no longer a list but an object.
+ */
+
+const {
+  extends: airbnbRules,
+  ...airbnbConfig
+} = require('eslint-config-airbnb-base');
+const importPlugin = require('eslint-plugin-import');
+const globals = require('globals');
+
+const mappedEnvs = {
+  es6: 'es2015',
```

Comment:
we should probably have an exhaustive mapping here just in case. does eslint expose it at all?

Reply:
The mapping is exhaustive: all the accepted eslint `env` values are the same except `es6`.
`es2016`, `es2018`, `es2019`, `es2022`, `es2023`, `es2024` are not in `globals`, but are not used by `eslint-config-airbnb` as for now (and `globals` has `es5` which is not accepted by `eslint`).

I will add a comment to make it clear.

Reply:
Indeed I can improve based on https://github.com/eslint/eslint/blob/main/conf/globals.js